portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,14 +47,21 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(messages);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    String message = messages.get((int) (Math.random() * messages.size()));
```

Comment:
This is slightly unclear. Is the goal to post a random message from your list to the server?

Reply:
yes, that's what it's doing. I added a comment to explain this. Is there a way to make the code itself more clear?

Reply:
This is fine. I've just gotten accustomed to using the [Random object ](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html) as all. lgtm